pkgs/tools/admin/manta/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib, buildNpmPackage, fetchurl, nodejs, installShellFiles, testers, node-manta }:
+
+let
+  source = builtins.fromJSON (builtins.readFile ./source.json);
+in
+buildNpmPackage rec {
+  pname = "manta";
+  inherit (source) version;
+
+  src = fetchurl {
+    url = "https://registry.npmjs.org/${pname}/-/${source.filename}";
+    hash = source.integrity;
+  };
+
+  npmDepsHash = source.deps;
+
+  dontNpmBuild = true;
```

Comment:
You can also just set `dontBuild = true`, if you don't need the default Makefile build phase to run.

Reply:
Fixed, thanks.